lib/git_reflow.rb

```diff
@@ -99,8 +99,10 @@ def deliver(options = {})
 
           if committed
             say "Merge complete!", :success
-            deploy_and_cleanup = ask "Would you like to push this branch to your remote repo and cleanup your feature branch? "
-            if deploy_and_cleanup =~ /^y/i
+            always_deploy_and_cleanup = GitReflow::Config.get('reflow.always-deploy-and-cleanup').strip == "true"
```

Comment:
`GitReflow::Config.get` will always call `strip` on the value it fetches from git config.